yearn/apy/common.py

```diff
@@ -6,10 +6,7 @@
 
 from yearn.utils import closest_block_after_timestamp
 
-SECONDS_PER_BLOCK = 13.25
-SECONDS_PER_YEAR = 31_536_000.0
-BLOCK_PER_DAY = (60 * 60 * 24) / SECONDS_PER_BLOCK
-
+SECONDS_PER_YEAR = 31_556_952.0
```

Comment:
this varies by vault version, 0.3.3 changed SECS_PER_YEAR from 31_557_600 to 31_556_952